index.js

```diff
@@ -34,7 +40,7 @@ app.listen(port, () => {
     console.log(`Server is running on port ${port}`); 
 });
 
-app.get("/", async (req, res) => {
-    await user.save(); 
+app.get("/", async(req, res) => { 
+    await user.save();
```

Comment:
Might it be better to move the user object here into the route, since this is where it's being saved?

Reply:
Done - this was extra to requirements and basically duplicating code.  Thanks! :)